python/cumulative.cpp

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2020 Scipp contributors (https://github.com/scipp)
+/// @file
+/// @author Simon Heybrock
+#include "pybind11.h"
+
+#include "scipp/variable/cumulative.h"
+
+using namespace scipp;
+using namespace scipp::variable;
+
+namespace py = pybind11;
+
+template <class T> void bind_cumsum(py::module &m) {
+  m.def(
+      "cumsum",
```

Comment:
The signature here: Use of bool is different from what is done elsewhere such as with `is_sorted` for example https://github.com/scipp/scipp/blob/master/python/operations.cpp#L76 where "ascending" is passed as a string not a bool.

Reply:
`mode='exclusive'`, or what would you suggest?

Reply:
Yes, that would fit with what we have above for `is_sorted`.

Reply:
Done.